test.ini.in

```diff
@@ -4,7 +4,6 @@ pyramid.includes =
     pyramid_mailer.testing
     pyramid_tm
 sqlalchemy.url = postgresql://{tests_db_user}:{tests_db_password}@{tests_db_host}:{tests_db_port}/{tests_db_name}
-sqlalchemy.echo = True
```

Comment:
Is this the line that throw all these undigestible lines in the tests output?

Reply:
yes, two hour of investigation :D

Reply:
The 2 best-used hours ever! I have not tested it yet but that would be soooooooo cool :) Thanks a lot!